.github/workflows/reusable-acceptance-tests.yml

```diff
@@ -79,13 +79,13 @@ jobs:
       run: npm run build
 
     - name: Start Docker environment
-      run: composer test:start
+      run: composer run test:start -- --quiet-pull
 
     - name: Log WordPress version
       run: docker compose run --rm wpcli core version
 
     - name: Run the tests
-      run: composer test:acceptance
+      run: composer run test:acceptance
```

Comment:
`composer test:acceptance` says `test:acceptance` is a Composer command.
`composer run test:acceptance` says `test:acceptance` is a Composer script.